executor/join.go

```diff
@@ -73,9 +73,9 @@ type HashJoinExec struct {
 	// execution, to avoid the concurrency of joiner.chk and joiner.selected.
 	joiners []joiner
 
-	probeChkResourceCh chan *probeChkResource
-	probeResultChs     []chan *chunk.Chunk
-	joinChkResourceCh  []chan *chunk.Chunk
+	probeChkResourceCh chan *chunk.Chunk
+	probeChkResultCh   chan *chunk.Chunk
+	joinChkResourceCh  chan *chunk.Chunk
```

Comment:
can you reuse the chunks like https://github.com/pingcap/tidb/pull/19178/ ?

Reply:
Seems chunk is already reused.